src/framework/theme/components/calendar-kit/components/calendar-week-number/calendar-week-number.component.ts

```diff
@@ -11,10 +11,12 @@ import { NbCalendarSize } from '../../model';
 @Component({
   selector: 'nb-calendar-week-numbers',
   template: `
-    <div class="sign">{{ weekNumberSymbol }}</div>
-    <div class="week-cell" *ngFor="let week of getWeeks()">{{ week }}</div>
+    <div class="sign-container">
+      <div class="sign">{{ weekNumberSymbol }}</div>
+    </div>
+    <div class="week-number" *ngFor="let week of getWeeks()">{{ week }}</div>
```

Comment:
Since you're already doing enhancements, why not to rewrite `getWeeks()` functionality? As I see it, `getWeeks()` result depends only on `weeks` input, so, don't you think it would be better to recalculate `getWeeks()` result in `weeks` setter and then just query it?

Reply:
Done in 6d55011424978aede484135f6b688218eea8c9e9. Implemented via `onChanges`. setter, getter require additional field. I believe `ngOnChanges` is more Angular way. What do you think?